---
layout: project
title: "Devloping high-throughput phenotyping technology"
handle: technology
image: /assets/images/papers/verticallogo.png
category: project
tags: [evolution]
---
{% include JB/setup %}

Quantifying the myriad phenotypic impacts of genetic change at the molecular, cellular and organismal levels, and doing so over and over in many different contexts requires incredibly high-throughput phenotyping methods. <b>We are developing several phenotyping technologies</b>, having thus far filed 3 patents, all of which list multiple lab members as inventors. Some of these methods pertain to new single-cell sequencing technology that allows combining DNA and RNA sequencing of the same cell. Other methods involve precisely quantifying the stability of thousands of protein variants at the same time. In addition to developing new methods, <b>we also implement, modify, and rigorously test</b> technologies created by other labs. For example, we have modified an inexpensive and highly multiplexable single-cell RNA sequencing platform for yeasts. We've also performed 79 technical replicates of a method to measure organismal fitness in order to determine how to achieve higher precision. <b>We care about reproducibility in science</b>, so we share the things we learn through <b>[open science campaigns on twitter like #1BigBatch](https://twitter.com/hashtag/1BigBatch?src=hashtag_click)</b>, preprints, and special issues on best practices. Many of the methods we use employ DNA barcodes, which are represented in our lab logo to the left.

<b>Relevant Papers:</b><br>
<b>-</b>Optimizing an existing single-cell RNA sequencing platform for yeasts <b>[link](https://kgslab.org/papers/paper/Split-Seq)</b><br>
<b>-</b>Designing an ultra-precise experiment to measure fitness <b>[link](https://kgslab.org/papers/paper/1bigbatch)</b><br>
<b>-</b>A special issue on Best Practices in Microbial Evolution (link coming soon)

